pkg/okteto/auth.go

```diff
@@ -136,34 +134,10 @@ func queryUser(ctx context.Context, client *graphql.Client, token string) (*q, e
 
 	req := getRequest(q, token)
 
-	if err := client.Run(ctx, req, &user); err != nil {
-		if err := client.Run(ctx, req, &user); err != nil {
-			if strings.Contains(err.Error(), "Cannot query field") {
-				log.Infof("query using the legacy parameters: %s", err)
-				return queryUserLegacy(ctx, client, token)
-			}
-
-			return nil, err
-		}
-	}
-
-	return &user, nil
-}
-
-func queryUserLegacy(ctx context.Context, client *graphql.Client, token string) (*q, error) {
```

Comment:
can we catch the error when we hit this (in the odd case that someone is running an older server) and tell them to upgrade the CLI?